pages/index.tsx

```diff
@@ -1,17 +1,144 @@
 import type { NextPage } from "next";
-import Head from "next/head";
-import styles from "../styles/Home.module.css";
+import Link from "next/link";
+import Card from "../components/Card";
+import Hazhar from '../components/Slider'
+import { ImFacebook } from "react-icons/im";
+import { ImTwitter } from "react-icons/im";
+import { ImGoogle } from "react-icons/im";
+
+
 
 const Home: NextPage = () => {
   return (
-    <div className={styles.container}>
-      <Head>
-        <title>Rekar</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+    <div className=" mx-auto mb-20 mt-5 md:container md:w-2/3 lg:w-2/3 ">
+      <nav className="flex items-center border border-none h-10 sm:w-screen md:container lg:container ">
+        <div className=" border border-violet-500 w-36 hover:bg-violet-100 h-8 ml-6 rounded-md text-center text-xs text-slate-500 pt-2 md:ml-16">
+          <Link href="/"> افتتاح حساب </Link>
+        </div>
+        <div className=" pl-5 text-violet-600 text-sm pb-1">
+          <Link href="/"> ورود </Link>
+        </div>
+        <div className="flex text-xs md:justify-around lg:ml-auto lg:mr-10">
+          <a href="/" className="p-1 sm:p-2">{" "}تماس با ما{" "}</a>
+          <a href="/" className="p-1 sm:p-2">{" "}چرا ریکار{" "}</a>
+          <a href="/" className="p-1 sm:p-2">{" "}درباره{" "}</a>
+          <a href="/" className="p-1 sm:p-2">{" "}خانه{" "}</a>
+        </div>
+        <img src="IMG/rekar-logotype.svg" alt="rekarlogotype" />
+      </nav>
+      <div className=" mt-16 flex flex-wrap justify-center md:flex-nowrap lg:flex-nowrap lg:ml-10 ">
+        <div className="border-2 border-none w-80 mr-7 text-center">
+          <img className=" mx-auto pt-14" src="IMG/rekar-logo.svg" />
```

Comment:
- Use <Image /> tag from next/image instead of using html <img /> in all of your project

Reply:
Error fixed